_rules/inline-link-in-paragraph-has-distinguishable-content-e7cc2b.md

```diff
@@ -0,0 +1,148 @@
+rule_type: atomic
+description: |
+  This rule checks that inline links have content that distinguishes them from the surrounding text
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+  previous authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], where the closest element (including itself) that generates an [inline box][] is part of a [line box][] that has [inline boxes][inline box] generated from an element containing [visible][] [text nodes][text node] that do not have a [semantic link][] as a [descendant][] in the [flat tree][].
+
+**Note:** The test targets are visible link elements that are part of a block of content that is displayed inline.
+
+## Expectation
+
+Each target element has content (such as an image or text) visually located inside, or immediately before or after the test target, that indicates the test target is a link.
+
+## Assumptions
+
+- This rule assumes that [`br` HTML elements][br] are only used for line breaks that are actually part of the content and not for separating thematic groups of the content. 
+- This rule assumes that the link is distinguishable from the rest of the text with color, which means it fails SC 1.4.1 when there is not another way to distinguish it.
+
+## Accessibility Support
+
+_No accessibility support issues known._
+
+## Background
+
+- [Understanding Success Criterion 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
+- [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](https://www.w3.org/WAI/WCAG21/Techniques/general/G183)
+- [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73)
+- [C15: Using CSS to change the presentation of a user interface component when it receives focus](https://www.w3.org/WAI/WCAG21/Techniques/css/C15)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This is a link that is a descendant of a paragraph element, and therefore in an inline block of content. An icon is added to distinguish it as a link.
+
+```html
+<style>
+	*,
+	a.test {
+		text-decoration: none;
+	}
+</style>
+<p>
+	Read about WAI on the
+	<a class="test" href="http://w3.org/WAI">WAI webpage <img src="/test-assets/be4d0c/icon.png" alt=""/></a>.
+</p>
+```
+
+#### Passed Example 2
+
+This is a link that is a descendant of a paragraph element, and therefore in an inline block of content. Text is added to distinguish it as a link.
+
+```html
+<style>
+	*,
+	a.test {
+		text-decoration: none;
+	}
+</style>
+<p>Read about WAI on the <a class="test" href="http://w3.org/WAI">WAI webpage by following this link</a>.</p>
```

Comment:
I don't think this passes. See my comment on the composite rule for details. The fact that we can both read this rule and come to different conclusions about this example tells me the rule's expectation needs to be updated too.

Reply:
I haven't really understood why you don't think this should pass (not even from your comments in the composite rule).
I don't understand what you mean with "visual means". The expectation requires some form of content that indicates the link is a link. The "visual" part is just to ensure that it is located (visually) close to the link.
When I read text stating "following this link" I expect at least the "link" word to be a link. So, there is content (the text) in the link indicating it is a link.

Could you further elaborate on why you don't think this passes?

Reply:
I would say text is a "visual means of conveying information", and thus this should pass.